recipes-ni/ni-hw-scripts/ni-hw-scripts-crio_1.0.bb

```diff
@@ -0,0 +1,54 @@
+SUMMARY = "Support scripts and utilities common to all NI hardware products"
+DESCRIPTION = "Support scripts and utilities for all NI hardware products which are supported by NI LinuxRT."
```

Comment:
Did you forget to update these for `-crio`?

Reply:
Doh!

Reply:
Updated for cRIO.